extensions/community_extensions/GPC.md

```diff
@@ -0,0 +1,38 @@
+### Global Privacy Control
+
+Issue: [#98](https://github.com/InteractiveAdvertisingBureau/openrtb/issues/98)
+
+The goal is to support passing the Global Privacy Control signal to downstream participants in order to allow them to make decisions on how to interpret the privacy state of a user in regard to applicable regulations. 
+
+Downstream consumers of this signal should use it as the primary signal where they believe it applies, since not all publishers may use the signal to change their interpretation of other privacy signals or may interpret it the same way as a downstream consumer. So, where the downstream consumer of OpenRTB signals sees both a USPAPI signal and a GPC signal on a request and considers the GPC signal to mean an opt-out in California, and where the USPAPI signal does not indicate such an opt-out, then the downstream consumer should consider the user opted-out.
```

Comment:
While this may be true, I'm not sure it's the appropriate place to state it

Reply:
@patmmccann Is there a better place to open up such a discussion?

Reply:
I'm not sure, but this looks like legal advice to me

Reply:
+1 I expect we have to strike this paragraph. I recommend following the pattern for our solutions to other regulations, e.g., https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md.

Reply:
>...should use it as the primary signal...

Is this dictate appropriate? What does "primary signal" mean? @wittjill's [earlier suggestion](https://github.com/InteractiveAdvertisingBureau/openrtb/pull/99#pullrequestreview-992612321) we scope our purpose to facilitation was spot on

>Can we change the two intro paragraphs so that it is more in line with how other privacy specs are written, something like:
>
>>This community extension supports passing the Global Privacy Control signal to downstream participants in order to allow them to make decisions on how to interpret the privacy state of a user in regard to applicable regulations.
>>
>>For more information about the Global Privacy Control specification visit: ​​https://globalprivacycontrol.org/


I'd propose this further minimalisation:

>This community extension supports passing the Global Privacy Control signal that downstream recipients may use to determine how to process the request. 
>
>For more information about the Global Privacy Control specification visit: ​​https://globalprivacycontrol.org/